internal/mcpclient/transport.go

```
@@ -0,0 +1,64 @@
+	}
+	return nil
+}
+
+func (c *MCPServerConfig) newTransport() mcp.Transport {
+	switch c.Transport {
+	case "streamable_http":
+		return &mcp.StreamableClientTransport{
+			Endpoint: c.URL,
+		}
+	case "stdio":
+		cmd := exec.Command(c.Command, c.Args...)
```

golangci-lint (2.12.1)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)